setup.py

```diff
@@ -84,16 +84,16 @@ def run(self):
                     os.symlink(rel_script, sym_name)
                     newoutfile = os.path.join(rel_script_dir, sym_name)
                     self.outfiles.append(newoutfile)
-                    log.info("symlink %s to %s newoutfile %s" % (rel_script, sym_name, newoutfile))
+                    log.info("symlink %s to %s newoutfile %s", rel_script, sym_name, newoutfile)
```

Comment:
this is not fancylogger or logging module. make sure you have tested this (e.g. look at the  messages when doing a `python setup.py install --user`)